examples/assessors/README.md

```diff
@@ -2,10 +2,10 @@
 
 *Assessor receive intermediate result from Trial and decide whether the Trial should be killed. Once the Trial experiment meets the early stop conditions, the assessor will kill the Trial.*
 
-So, if user want to implement a customized Assessor, she/he only need to:
+So, if want to implement a customized Assessor, you only need to:

```

Comment:
how about "if users want to ...., they only need to:"?

Reply:
sounds good. Will update